<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Source image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Thresholded image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Result image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="PiiImageDisplay" name="_pSourceImageDisplay" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="PiiImageDisplay" name="_pThresholdedImageDisplay" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Source image</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="_pSourceImageName">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_pBrowseImageButton">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Thresholding</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Threshold:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="_pThreshold">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="_pInverseThreshold">
         <property name="text">
          <string>Inverse</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Morphology</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="_pMorphologyTypeCombo">
         <item>
          <property name="text">
           <string>Erode</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dilate</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Open</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Close</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tophat</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bottomhat</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="_pMorphologySize">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="PiiImageDisplay" name="_pResultImageDisplay" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="_pActionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="_pActionSelectImages">
   <property name="text">
    <string>Select images...</string>
   </property>
  </action>
  <action name="_pActionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEka">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Eka</string>
   </property>
  </action>
  <action name="actionToka">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toka</string>
   </property>
  </action>
  <action name="actionKolmas">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kolmas</string>
   </property>
  </action>
  <action name="actionNelj_s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nelj√§s</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PiiImageDisplay</class>
   <extends>QWidget</extends>
   <header location="global">PiiImageDisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_pActionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
